tests/rt.sh

```diff
@@ -575,7 +590,7 @@ fi
 if [[ $ECFLOW == true ]]; then
 
   # Default maximum number of compile and run jobs
-  MAX_BUILDS=10
+  MAX_BUILDS=30
```

Comment:
Until we move to oneapi, I think our licenses are limited for Intel, at least on WCOSS2 this is an issue as the builds will sit waiting for a license to open up, and thus timeout. I am not sure if other machines have this.

Reply:
Reverted back to 10.